readthedocs/projects/views/private.py

```diff
@@ -113,6 +114,28 @@ def get_context_data(self, **kwargs):
             # Alternatively, dynamically override super()-derived `project_list` context_data
             # context[self.get_context_object_name(filter.qs)] = filter.qs
 
+            projects = AdminPermission.projects(user=self.request.user, admin=True)
+            if (timezone.now() - self.request.user.date_joined).days < 30:
```

Comment:
Instead of giving new users this content for 30 days unconditionally, perhaps we want more conditional logic here. For example, once a user has a project they might benefit more from pull request builds than example projects. I don't know the exact balance here.

Maybe a shorter time frame than 30 days? Maybe we only show this if the user has not added a project yet?

Reply:
Yeah, I'm not settled on this logic either. I just want to have _something to start with_, but this can definitely be improved

> Maybe a shorter time frame than 30 days? Maybe we only show this if the user has not added a project yet?

I think I want to show this _even_ if the user has already 1 project imported because they are still testing our platform. I'm leaning towards showing it during 7 days after they have imported their first project, or something similar.

Reply:
That seems like a good compromise, I agree it's probably mostly the timeframe and it makes sense to show this even after a project is created.